Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule_test.py

```diff
@@ -634,6 +634,39 @@ def test_update_last_modified_indicator_date(self, last_modifies_client, last_mo
 
         assert mock_client.last_fetched_indicator__modified == expected_modified_result
 
+    @pytest.mark.parametrize(
+        'objects_to_fetch_param', ([], ['dummy'])
```

Comment:
I would add a case for 2 objects, but thats up to you.

Reply:
Added case for 2 objects.